app/views/articles/manage.html.erb

```diff
@@ -11,7 +11,6 @@
         <h3 class="manage-page-success"><%= flash[:pins_success] %></h3>
       <% end %>
       <h2>Tools:</h2>
-      <p><strong>Suggest a Tweet:</strong> Help get your post in front of more <%= community_members_label %> by suggesting a tweet that <a href="https://twitter.com/<%= SiteConfig.social_media_handles["twitter"] %>">@<%= SiteConfig.social_media_handles["twitter"] %></a> editors can use.
       <p><strong>Experience Level of Post:</strong> Is your post written more for a beginner or an advanced audience?  Adjust this level as a <em>gentle</em> indicator which will help show the post to the people who will benefit the most.
       <h2>Tips:</h2>
       <ol>
```

Comment:
Should we remove the copy on line `19` (now line `18` after the removal of the Tweet copy that was on line `14`) that is shown below?:

> Write your own tweet about the post, describing personally why you wrote it or why people might find it useful. We may retweet you.

Reply:
Nope, that's not related to the feature that was removed. We still want to encourage people to tweet their own content from their own Twitter accounts.

Reply:
Got it! Thanks so much for the clarification, @mstruve! 😊

Reply:
Ah, one always slips through the cracks...